packages/@aws-cdk/aws-codepipeline-actions/test/cloudformation/cloudformation-stackset-pipeline-actions.test.ts

```diff
@@ -0,0 +1,381 @@
+import { Match, Template } from '@aws-cdk/assertions';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import * as cpactions from '../../lib';
+import { TestFixture } from './TestFixture';
+/* eslint-disable quote-props */
+
+let stack: TestFixture;
+let importedAdminRole: iam.IRole;
+beforeEach(() => {
+  stack = new TestFixture();
+  importedAdminRole = iam.Role.fromRoleArn(stack, 'ChangeSetRole', 'arn:aws:iam::1234:role/ImportedAdminRole');
+});
+
+describe('StackSetAction', () => {
+  function defaultOpts() {
```

Comment:
Any chance for a return type here please? 😇

Reply:
It will fuck with TypeScript's type inference.